engines/ags/globals.h

```diff
@@ -1092,6 +1092,9 @@ class Globals {
 	std::set<String> _tellInfoKeys;
 	int _loadSaveGameOnStartup = -1;
 
+	// ScummVM GUIO-controlled flag to save a screenshot
+	// when saving (used for saves thumbnails)
+	bool _SaveThumbnail = true;
```

Comment:
```suggestion
	bool _saveThumbnail = true;
```
(and this obviously has to be changed where it is set and used as well)

https://wiki.scummvm.org/index.php?title=Code_Formatting_Conventions#Naming

Reply:
Changed the capitalization, should be fine now